### PR TITLE
New output directory structure implemented under data_products

### PR DESCRIPTION
**Pull request by felipeji**
This branch contain a new structure for the outputs of the pipeline organised now under the `data_products` directory that the pipeline creates in the reduction process. The reduced data cubes now have the suffix `cube` instead of `p11`.
I also  run `black` formater over `reduce_data.py`.

### CODE REVIEW

File: reduction_scripts/reduce_data.py [hunk 1]
```diff
@@ -779,91 +882,103 @@ def run_save_3dcube(metadata, prev_suffix, curr_suffix, **args):
...
             else:
                 pass
 
-        #------------------------------------------------------------------------
-        #------------------------------------------------------------------------
+    # ----------------------------------------------------------
+    # Move reduce cube to the data_products directory
+
+    destination_dir = "./data_products/"
+
```
Comment (CMartinezLombilla) [inline reduction_scripts/reduce_data.py:966]: Please, define `destination_dir` using `working_dir` so the pipeline doesn't rely on any assumption.

File: reduction_scripts/reduce_data.py [hunk 2]
```diff
@@ -779,91 +882,103 @@ def run_save_3dcube(metadata, prev_suffix, curr_suffix, **args):
...
+    # Red
+    red_cubes_path = glob.glob("./data_products/intermediate/red/*.cube.fits")
+    for red_cube_path in red_cubes_path:
+        shutil.move(red_cube_path, destination_dir)
+
+    # Blue
+    blue_cubes_path = glob.glob("./data_products/intermediate/blue/*.cube.fits")
+    for blue_cube_path in blue_cubes_path:
+        shutil.move(blue_cube_path, destination_dir)
+
```
Comment (CMartinezLombilla) [inline reduction_scripts/reduce_data.py:976]: The same for the lists `red_cubes_path` and `blue_cubes_path`, please use `working_dir`.

File: reduction_scripts/reduce_data.py [hunk 3]
```diff
@@ -779,91 +882,103 @@ def run_save_3dcube(metadata, prev_suffix, curr_suffix, **args):
...
     for arm in obs_metadatas.keys():
-        #------------------------------------------------------------------------
-        #      LOAD JSON FILE WITH USER REDUCTION SETUP             
-        #------------------------------------------------------------------------
+        # ------------------------------------------------------------------------
+        #      LOAD JSON FILE WITH USER REDUCTION SETUP
+        # ------------------------------------------------------------------------
         obs_metadata = obs_metadatas[arm]
         reduction_scripts_dir = os.path.dirname(__file__)
         working_dir = os.getcwd()
```
Comment (CMartinezLombilla) [inline reduction_scripts/reduce_data.py:892]: Maybe is a good idea to put `reduction_scripts_dir` and `working_dir` before the loop?

**Review by CMartinezLombilla — COMMENTED** [review]
All good in general! Just amend what I mentioned in the comments about the directory variables. Then, feel free to merge.

File: reduction_scripts/reduce_data.py (hunk 3 above)
Comment (felipeji) [reply reduction_scripts/reduce_data.py:892]: done!

**Pull request merged**